### PR TITLE
DOC-6869 Release Notes for 2.7.4

### PR DESCRIPTION
**Pull request by ibsoln**
https://issues.couchbase.com/browse/DOC-6869

### CODE REVIEW

**Review by amarantha-k — APPROVED** [review]
lgtm. Will wait for Dan's approval before merging. Plan to publish on 10/20

File: modules/ROOT/pages/release-notes.adoc
Comment (djpongh) [inline modules/ROOT/pages/release-notes.adoc]: Remove extra spaces

Comment (ibsoln) [reply modules/ROOT/pages/release-notes.adoc]: Thanks @djpongh, resolved in commit aac8fbace52f692bbb39e4095dd26cc578cb4709

**Review by amarantha-k — APPROVED** [review]

**Pull request merged**
